spring-webserver/server-java/src/main/java/net/corda/server/Server.java

```diff
@@ -0,0 +1,21 @@
+package net.corda.server;
+
+import org.springframework.boot.Banner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * Our Spring Boot application.
+ */
+@SpringBootApplication
+public class Server {
+    /**
+     * Starts our Spring Boot application.
+     */
+    public static void main(String[] args) {
+        SpringApplication app = new SpringApplication(Server.class);
+        app.setBannerMode(Banner.Mode.OFF);
+        app.setWebEnvironment(true);
```

Comment:
setWebEnvironment is depricated. 
Lets use setWebApplicationType instead

Reply:
changed and tested.